modules/roles_profiles/manifests/profiles/cltbld_user.pp

```diff
@@ -2,8 +2,11 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
-class roles_profiles::profiles::cltbld_user {
-
+class roles_profiles::profiles::cltbld_user (
```

Comment:
As a rule of thumb, we should not be passing parameters to any profiles.  I intend to fix every profile that does this currently.  Although, since provides defaults that align with existing settings, I'm inclined to let this apply and fix it later when we fully implement role hiera data.

Reply:
I'm okay with removing this. It was leftover from loaner testing and isn't used by the builders.